src/code42cli/cmds/alerts.py

```diff
@@ -283,3 +297,76 @@ def _get_alert_extractor(sdk, handlers):
 
 def _get_alert_cursor_store(profile_name):
     return AlertCursorStore(profile_name)
+
+
+@alerts.command()
+@opt.sdk_options()
+@alert_id_arg
+def show(state, alert_id):
+    """Display the details of a single alert."""
+    formatter = OutputFormatter(OutputFormat.TABLE, _get_default_output_header())
```

Comment:
Would it make sense to include the observations in the `show` output? Maybe pretty-print the json for readability?

Reply:
Since the observations blob is quite long, perhaps a flag `--include-observations` is warranted. Similarly to how we have `--include-file-events` for `cases show`